proposed/links.md

```diff
@@ -194,3 +291,27 @@ interface LinkCollectionInterface
     public function getLinksByRel($rel);
 }
 ~~~
+
+#### 3.3 `Psr\Http\Link\EvolvableLinkCollectionInterface`
+
+~~~php
+<?php
+
+namespace Psr\Http\Link;
+
+/**
+ * An evolvable link collection value object.
+ */
+interface EvolvableLinkCollectionInterface extends LinkCollectionInterface
+{
+    /**
+     * Returns an instance with the specified link included.
+     *
+     * @param LinkInterface $link
+     *   A link object that should be included in this collection.
+     *
```

Comment:
Remove the extra empty line. Also, IIRC, lines wrapped from `@param/@return` notations are supposed to be four spaces, no?